bikeshop_project/assets/js/components/Purpose.jsx

```diff
@@ -14,6 +14,9 @@ const Purpose = ({ initialValue, handleChange }) => (
     <MenuItem value={'WORKSHOP'} primaryText="Workshop" />
     <MenuItem value={'DONATE'} primaryText="Donate" />
     <MenuItem value={'STAFF'} primaryText="Staff" />
+    <MenuItem value={'PARTS'} primaryText="Parts" />
+    <MenuItem value={'BUY_BIKE'} primaryText="Buy Bike" />
+    <MenuItem value={'TOUR / VISIT'} primaryText="Tour / Visit" />
```

Comment:
Is this the right value? I kinda forget how this works... I have this feeling it's going to be `'TOUR'`